marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/jobs.sjs

```diff
@@ -173,7 +173,7 @@ class Jobs {
      }, this.config.JOBDATABASE));
   }
 
-  updateBatch(jobId, batchId, batchStatus, uris) {
+  updateBatch(jobId, batchId, batchStatus, uris, stackTrace) {
```

Comment:
The error object is optionally being passed, which has the stackTrace array property.
```suggestion
  updateBatch(jobId, batchId, batchStatus, uris, error) {
```